### PR TITLE
add self service example project

### PR DESCRIPTION
**Pull request by mastudillot**
Added self service example project

### CODE REVIEW

File: TransbankAutoservicioSDKConsoleExample/Program.cs [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+            Console.WriteLine("\n");
+            bool print = false;
+            bool voucher = false;
+            int amount = 0;
+            string ticket = "";
+            do
+            {
+                switch (key)
+                {
+                    case 1:
```
Comment (ffiebig) [inline TransbankAutoservicioSDKConsoleExample/Program.cs:32]: It's not necessary but I would appreciate a few comments on conditions which involve only numbers. What I mean is a comment saying the text of the menu item or a small explanation about it

File: TransbankAutoservicioSDKConsoleExample/Program.cs [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+                            voucher = Console.ReadKey().KeyChar == '1';
+                            Console.WriteLine();
+                            Task<LastSaleResponse> response = POSAutoservicio.Instance.LastSale(voucher);
+                            response.Wait();
+                            Console.WriteLine(response.Result);
+                            break;
+                        }
+                        catch (Exception e)
+                        {
+                            Console.WriteLine("Error: " + e.Message);
```
Comment (ffiebig) [inline TransbankAutoservicioSDKConsoleExample/Program.cs:147]: Maybe use blocks for repetitive code like this one I mean the try catch reutilization. Something like what we use in the Webpay SDK https://github.com/TransbankDevelopers/transbank-sdk-dotnet/blob/master/Transbank/Common/ExceptionHandler.cs

**Review by ffiebig — DISMISSED** [review]
I left a few comments, it's not necessary to implement them. It looks good

File: TransbankAutoservicioSDKConsoleExample/Program.cs (hunk 1 above)
Comment (mastudillot) [reply TransbankAutoservicioSDKConsoleExample/Program.cs:32]: The showmenu function has the description of switch / case

File: TransbankAutoservicioSDKConsoleExample/Program.cs (hunk 2 above)
Comment (mastudillot) [reply TransbankAutoservicioSDKConsoleExample/Program.cs:147]: 👌

**Review by ffiebig — APPROVED** [review]

**Pull request merged**